detection_rules/misc.py

```diff
@@ -213,19 +213,6 @@ def parse_config():
     return _CONFIG
 
 
-def set_param_values(ctx, param, value):
-    """Get value for defined key."""
-    key = param.name
-    config = parse_config()
-    env_key = 'DR_' + key.upper()
-    prompt = True if param.hide_input is not False else False
-
-    if value:
-        return value
-    elif os.environ.get(env_key):
-        return os.environ[env_key]
-    elif config.get(key) is not None:
-        return config[key]
-    elif prompt:
-        return click.prompt(key, default=param.default if not param.default else None, hide_input=param.hide_input,
-                            show_default=True if param.default else False)
+def getenv(name):
```

Comment:
I like what this is intended to solve, but this totally eliminates the ability to use config files for anything.

Reply:
7165756e5ba42e06aa8ed7b6bc48225cc9552ac2